packages/flutter/test/services/fake_platform_views.dart

```diff
@@ -207,6 +262,31 @@ class FakeMotionEvent {
 
   @override
   String toString() {
-    return 'FakeMotionEvent(action: $action, pointerIds: $pointerIds, pointers: $pointers)';
+    return 'FakeAndroidMotionEvent(action: $action, pointerIds: $pointerIds, pointers: $pointers)';
+  }
+}
+
+class FakeIosPlatformView {
+
+  FakeIosPlatformView(this.id, this.type);
+
+  final int id;
+  final String type;
+
+  @override
+  bool operator ==(dynamic other) {
+    if (other is! FakeIosPlatformView)
```

Comment:
compare runtimeType instead of using `is` in `operator==`, otherwise subclasses will have asymmetric `==` operators

Reply:
done (here and in FakeAndroidPlatformView)